ccm_web/client/tsconfig.json

```diff
@@ -1,26 +1,15 @@
 {
+  "extends": "../tsconfig.json",
```

Comment:
does this means this is extending all the rules from the root `tsconfig.json`?

Reply:
Yes, that is my understanding. Any repeated properties in this `tsconfig.json` will override what's in the base config.

https://www.typescriptlang.org/tsconfig#extends